app/assets/stylesheets/modules/admin.scss

```diff
@@ -36,9 +36,24 @@
         background-color: $white;
       }
     }
+
+    td.active.day {
+      background-color: $color-pantone-403;
+    }
   }
 }
 
 .dashboard-table {
   text-align: center;
 }
+
+.editable-date {
+  .icon-arrow-left {
+    @extend .glyphicon;
+    @extend .glyphicon-arrow-left;
+  }
```

Comment:
is the `.icon-arrow-left` block what hound is complaining about?  seems like the line below is empty, so not sure what to do here...